### PR TITLE
testing/conmon: new aports

### PR DESCRIPTION
**Pull request by kohnish**
Add conmon as a preparation for adding podman to alpine testing repository.

### CODE REVIEW

File: testing/conmon/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+depends=""
+makedepends="glib-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/containers/${pkgname}/archive/v${pkgver}.tar.gz"
+builddir="$srcdir/${pkgname}-${pkgver}"
+subpackages="$pkgname-doc"
+
+build() {
+        PREFIX=/usr make
+        :
```
Comment (andypost) [inline testing/conmon/APKBUILD]: looks like typo

File: testing/conmon/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Contributor:
+# Maintainer:
+pkgname="conmon"
+pkgver="2.0.7"
+pkgrel=0
+pkgdesc="An OCI container runtime monitor."
+url="https://github.com/containers/conmon"
+arch="all"
+license="Apache License 2.0"
```
Comment (andypost) [inline testing/conmon/APKBUILD]: `Apache-2.0` according to https://spdx.org/licenses/

File: testing/conmon/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+depends=""
```
Comment (andypost) [inline testing/conmon/APKBUILD]: no reason for that line

File: testing/conmon/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+makedepends="glib-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/containers/${pkgname}/archive/v${pkgver}.tar.gz"
+builddir="$srcdir/${pkgname}-${pkgver}"
```
Comment (andypost) [inline testing/conmon/APKBUILD]: this is default could be skiped

File: testing/conmon/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+subpackages="$pkgname-doc"
```
Comment (andypost) [inline testing/conmon/APKBUILD]: already defined in line 12

**Comment by andypost** [comment]
Also please squash commits into single one

**Comment by kohnish** [comment]
> Also please squash commits into single one

Done. :)

**Pull request closed without merging**